src/pod.rs

```diff
@@ -97,26 +100,31 @@ unsafe impl<T: PodTransmutable> PodTransmutable for [T; 32] {}
 ///
 /// An error is raised in one of the following situations:
```

Comment:
In keeping with stdlib docs panics are "raised", but errors are "returned".
Occurs on:
 *  L101
 *  L136
 *  L174
 *  L225